### DISCUSSION TITLE
Help to create bash to compose 2 images with specific string for background and foreground

### DISCUSSION THREAD

**Discussion by ChritianK** [ImageMagick/ImageMagick#6279] [Help]
### ImageMagick version

7.0.8 12

### Operating system, version and so on

Win10 Enterprise

### Description

Hi All ! 
I'm very newbie scripting bash and nee your help to made a bash script to compose 2 images with Imagemagick, but I need to identify the background and foreground image.
Command should be this :
 magick convert image01_background.jpeg image01_foreground.jpeg -gravity center -composite image01_result.jpeg

How can I made a bash script to identify in the name of file the string contains "background" and "foreground" to compose correctly images ?
Have a lot of images to compose so a script will be the best efficient way to do it !
Thanks in advance for your help !
Best Regards, 
Christian

**Comment by snibgo** [comment]
This is an ImageMagick forum. The question seems to be a question about scripting in bash, not about ImageMagick.

I suggest using awk. An internet search for "awk string search" will provide many examples.
